bundles/model/org.eclipse.smarthome.model.item/META-INF/MANIFEST.MF

```diff
@@ -1,48 +1,50 @@
- org.eclipse.smarthome.model.lazygen;resolution:=optional,
- org.eclipse.xtext.xbase.lib
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Item Model
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-SymbolicName: org.eclipse.smarthome.model.item;singleton:=true
+Require-Bundle: org.eclipse.xtext,
+ org.eclipse.xtext.generator;resolution:=optional,
+ org.apache.commons.logging;resolution:=optional,
+ org.eclipse.emf.codegen.ecore;resolution:=optional,
+ org.eclipse.emf.mwe.utils;resolution:=optional,
+ org.eclipse.emf.mwe2.launch;resolution:=optional,
+ com.ibm.icu;resolution:=optional,
+ org.eclipse.xtext.util,
+ org.eclipse.emf.ecore;visibility:=reexport,
+ org.eclipse.emf.common,
+ org.antlr.runtime,
+ org.eclipse.xtext.common.types,
+ org.eclipse.smarthome.model.lazygen;resolution:=optional,
+ org.eclipse.xtext.xbase.lib,
+ org.eclipse.xtend.lib
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Export-Package: org.eclipse.smarthome.model,
- org.eclipse.smarthome.model.formatting,
- org.eclipse.smarthome.model.generator,
- org.eclipse.smarthome.model.item,
- org.eclipse.smarthome.model.items,
- org.eclipse.smarthome.model.items.impl,
- org.eclipse.smarthome.model.items.util,
- org.eclipse.smarthome.model.parser.antlr,
- org.eclipse.smarthome.model.parser.antlr.internal,
- org.eclipse.smarthome.model.scoping,
- org.eclipse.smarthome.model.serializer,
- org.eclipse.smarthome.model.services,
- org.eclipse.smarthome.model.validation
-Import-Package: org.apache.commons.lang,
- org.apache.log4j,
- org.eclipse.smarthome.core.autoupdate,
- org.eclipse.smarthome.core.common.registry,
- org.eclipse.smarthome.core.items,
- org.eclipse.smarthome.core.items.dto,
- org.eclipse.smarthome.core.library.items,
- org.eclipse.smarthome.core.library.types,
- org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.model.core,
- org.eclipse.xtext.xbase.lib,
- org.osgi.framework,
- org.slf4j
-Service-Component: OSGI-INF/*.xml
+Export-Package: org.eclipse.smarthome.model,
+ org.eclipse.smarthome.model.formatting,
+ org.eclipse.smarthome.model.generator,
+ org.eclipse.smarthome.model.item,
+ org.eclipse.smarthome.model.items,
+ org.eclipse.smarthome.model.items.impl,
+ org.eclipse.smarthome.model.items.util,
+ org.eclipse.smarthome.model.parser.antlr,
+ org.eclipse.smarthome.model.parser.antlr.internal,
+ org.eclipse.smarthome.model.scoping,
+ org.eclipse.smarthome.model.serializer,
+ org.eclipse.smarthome.model.services,
+ org.eclipse.smarthome.model.validation
+Import-Package: org.apache.commons.lang,
+ org.apache.log4j,
+ org.eclipse.smarthome.core.autoupdate,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.items,
+ org.eclipse.smarthome.core.items.dto,
+ org.eclipse.smarthome.core.library.items,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.model.core,
+ org.eclipse.xtend.lib.annotations,
```

Comment:
This is a new dependency. Do we really need it (mandatorily) or only optional for the IDE to be able to resolve annotations?